localstack/utils/cloudformation/template_deployer.py

```diff
@@ -1120,6 +1129,9 @@ def execute_resource_action(resource_id, resources, stack_name, action_name):
         LOG.warning('Action "%s" for resource type %s not yet implemented' % (action_name, resource_type))
         return
 
+    if resource.get('Type') == 'AWS::DynamoDB::Table':
```

Comment:
I think we should try to keep this function clean of resource type specific fixes - these fixes should ideally be kept in a separate function. Wondering if we can add it to the `add_default_resource_props(..)` function - would it work if we add it here: https://github.com/localstack/localstack/blob/master/localstack/services/cloudformation/cloudformation_starter.py#L193